tests/test_callback_utils.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from parameterized import parameterized
+
+# from pytorch_lightning import Trainer
+from uetai.logger import SummaryWriter
+from uetai.callbacks.utils import check_logger
+
+
+class TestCallbackUtils(unittest.TestCase):
+    @parameterized.expand([
+        (None, False),
+        (SummaryWriter('uetai'), True),
+        (SummaryWriter('uetai', log_tool='tensorboard'), False, UserWarning),
+    ])
+    def test_check_logger(self, logger, available, expect=None):
+        log = check_logger(logger)
+        if expect is not None:
+            self.assertWarns(expect)
+        self.assertEqual(log, available)
+
+    # def test_finish_wandb_run(self):
```

Comment:
remove unused code

Reply:
I had a problem here. The function `train_finish_run` should be tested but while testing by that `test_finish_wandb_run`, the wandb backend also stoped other unrelated running SummaryWriter logger those init with wandb